### PR TITLE
Updating README with Error Codes and Minimum Rust Version

### PR DESCRIPTION
**Pull request by ACStone-MTS**
Included in this PR:

- Error Codes
- Specifying the necessary Rust Version

### CODE REVIEW

**Review by darthsiroftardis — APPROVED** [review]
Good

File: README.md [hunk 1]
```diff
@@ -489,3 +493,113 @@ ensures that as new modalities are added, and current modalities are extended, n
...
+|  79   | MissingContractWhiteList          |
+|  80   | InvalidContractWhitelist          |
+|  81   | UnlistedContractHash              |
+|  82   | InvalidContract                   |
+|  83   | EmptyContractWhitelist            |
+|  84   | MissingReceiptName                |
+|  85   | InvalidReceiptName                |
+|  86   | InvalidJsonMetadata               |
+|  87   | InvalidJsonFormat                 |
+|  88   | FailedToParseCep99Metadata        |
```
Comment (davidatwhiletrue) [inline README.md]: Is CEP99 correct here?

Comment (ACStone-MTS) [reply README.md]: I pulled them from https://github.com/casper-ecosystem/cep-78-enhanced-nft/blob/dev/contract/src/error.rs - so we'll need to fix it there as well if they're incorrect. @darthsiroftardis

Comment (darthsiroftardis) [reply README.md]: Yeah, thats a mistake thanks for the catch there @davidatwhiletrue

Comment (ACStone-MTS) [reply README.md]: Fixed.

**Pull request merged**